### DISCUSSION TITLE
Albanian language support

### DISCUSSION THREAD

**Discussion by e-jajaga** [openai/whisper#675] [Q&A] [answered]
Dear all,

You have some results for Albanian language. However, in the notebook example Multilingual_ASR.ipynb the langauage code is missing. 
`languages = {"af_za": "Afrikaans", ...`
What code have you used for Abanian langauage settings?  Is it `sq_al`?

**Comment by jongwook** [comment, marked as answer]
Those language codes are the ones used by the [Fleurs dataset](https://huggingface.co/datasets/google/fleurs), which did not include Albanian. Those are slightly different from what's used in our [tokenizer.py](https://github.com/openai/whisper/blob/main/whisper/tokenizer.py).

In Whisper, you can specify `--language Albanian` or `--language sq` to tell the model to transcribe Albanian speech, but we unfortunately don't have quantitative metrics on Whisper's performance in Albanian. I'd be curious to know if you can share your experience.

**Reply by rockyalb** [reply]
Hi there, I am Albanian and was messing around with Whisper. I was just looking for some others feedback before posting mine but as far as Albanian is concerned even using source audio from news channels which should be easiest to transcribe I hate to say that the results are unusable. 

Whisper kind of tries to type what it hears but it does not seem to have a good vocabulary to choose the best guess from and it just creates random words and mixes words together.

**Reply by jongwook** [reply]
Thanks for sharing your experience!

**Reply by marisbasha** [reply]
I have created a fine-tuned version for Albanian. 

https://huggingface.co/spaces/n-iv/wsq 

If anyone is interested in collaborating or using the model message me.

**Reply by florijanqosja** [reply]
@marisbasha please can u conact me https://www.linkedin.com/in/florijan-qosja/
I have many models as well. and i am building a new dataset

**Reply by carlos-havier** [reply]
@marisbasha I am quite interested in using your fined-tuned version for Albanian. Is it possible to access it? Thanks

**Reply by marisbasha** [reply]
https://huggingface.co/niv-al/peshperima-large-v2

**Reply by carlos-havier** [reply]
> https://huggingface.co/niv-al/peshperima-large-v2

@marisbasha Thanks! Is the finetuning data available? Was interested in a pure transformer model, so I can use it with CTranslate2 :)
